### DISCUSSION TITLE
Rendering of chart is off. Any ideas ?.

### DISCUSSION THREAD

**Discussion by traderblakeq** [kernc/backtesting.py#797] [General]
```
`from backtesting import Backtest, Strategy
from backtesting.lib import crossover

from backtesting.test import SMA, GOOG


class SmaCross(Strategy):
    def init(self):
        price = self.data.Close
        self.ma1 = self.I(SMA, price, 10)
        self.ma2 = self.I(SMA, price, 20)

    def next(self):
        if crossover(self.ma1, self.ma2):
            self.buy()
        elif crossover(self.ma2, self.ma1):
            self.sell()


bt = Backtest(GOOG, SmaCross, commission=.002,
              exclusive_orders=True)



stats = bt.run()
print(stats)
bt.plot()`
```

![image](https://user-images.githubusercontent.com/36839989/199945352-66e105a9-a181-4675-8998-16cb48d16d12.png)

![image](https://user-images.githubusercontent.com/36839989/199996895-56bf49c6-a25a-4e6a-8d60-7718f0842d53.png)

![image](https://user-images.githubusercontent.com/36839989/199985243-f2625e18-3071-4293-8936-fec059f5732b.png)

backtesting version: 0.3.3
python verson: 3.10.6
OS: Linux pop 22.04

Did try in both Firefox & Chrome.

**Comment by kernc** [comment]
This is a current issue with `bokeh>=3.0` and [`sizing_mode='stretch_width'`](https://docs.bokeh.org/en/dev-3.0/docs/examples/basic/layouts/sizing_mode.html) I haven't yet been able to resolve.

**Reply by traderblakeq** [reply]
Would if fix the issue to downgrade bokeh to 2.4.3 ?.

**Reply by kernc** [reply]
It would temporarily _work around it_. 😅 `pip install -U bokeh==2.4.3`

**Reply by traderblakeq** [reply]
Thanks did fix it. 

If and when you resolve, you could give a heads up in here :D

**Comment by jiiins** [comment]
I'm not sure if it helps, but this is the error:
`BokehDeprecationWarning: Passing lists of formats for DatetimeTickFormatter scales was deprecated in Bokeh 3.0. Configure a single string format for each scale
D:\Dropbox\_python\2\venv\lib\site-packages\backtesting\_plotting.py:250: UserWarning: DatetimeFormatter scales now only accept a single format. Using the first provided: '%d %b'
  formatter=DatetimeTickFormatter(days=['%d %b', '%a %d'],
`
